rules/integrations/cloud_defend/execution_interactive_shell_spawned_from_inside_a_container.toml

```diff
@@ -41,8 +41,8 @@ process where container.id: "*" and
 event.type== "start" and
 
 /*D4C consolidates closely spawned event.actions, this excludes end actions to only capture ongoing processes*/
-event.action in ("fork", "exec") and event.action != "end"
```

Comment:
Not sure if the Event aggregation affects Defend for Cloud, but I'm removing `event.action != "end"` as it would never match anyway as `event.action in ("fork", "exec")` is specified